app/src/main/java/com/mathewsachin/fategrandautomata/ui/skill_maker/SkillMakerKukulcanTarget.kt

```diff
@@ -0,0 +1,64 @@
+package com.mathewsachin.fategrandautomata.ui.skill_maker
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mathewsachin.fategrandautomata.R
+import com.mathewsachin.fategrandautomata.scripts.models.ServantTarget
+
+@Composable
+fun SkillMakerKukulcanTarget(
```

Comment:
Since they are closely related, I moved this function to the same file as SkillMakerKukulcan.